### ISSUE TITLE
extra slash in path

### ISSUE THREAD

**Issue by agilityvision** [iosdevzone/IDZSwiftCommonCrypto#66]
Current module maps generate a path with what seems to be an extra slash after CommonCrypto

```
module CommonCrypto [system] {
  header "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator10.1.sdk/usr/include/CommonCrypto//CommonCrypto.h"
  header "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator10.1.sdk/usr/include/CommonCrypto//CommonRandom.h"
  export *
}
```

**Comment by iosdevzone** [comment]
This should be fixed in 159744d72fcb3f20f67821b78bfe6a9e52be1635
As soon as Travis passes I'll close this out and it will go out in the next release.
Thanks for pointing this out!

**Comment by iosdevzone** [comment]
Travis has passed so closing this.
Since this is more a benign "lack of attention to detail" issue rather than a bug, I will not be pushing a release for it. It will be contained in the next release.
